packages/babel-plugin/src/styled/__tests__/call-expression.test.ts

```diff
@@ -538,4 +538,87 @@ describe('styled object call expression', () => {
 
     expect(actual).toIncludeMultiple(['color:black', 'font-weight:400', 'background-color:white']);
   });
+
+  it('should transform variable in a nested template literal', () => {
+    // this is the output of applying @atlaskit/tokens babel plugin
+    // (as of v1.0.0) to some code similar to this:
+    //     styled.div({
```

Comment:
Thanks for providing what looks like before applying `@atlaskit/tokens` 👍. That really helps me understand the code.